core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -346,7 +346,10 @@ func (l *logPollerWrapper) checkForRouteUpdates() {
 			l.lggr.Errorw("LogPollerWrapper: error calling getCurrentCoordinators", "err", err)
 			return
 		}
-		l.handleRouteUpdate(active, proposed)
+
+		handleRouteCtx, handleRouteCancel := utils.ContextFromChan(l.stopCh)
+		defer handleRouteCancel()
+		l.handleRouteUpdate(handleRouteCtx, active, proposed)
```

Comment:
I think we can do some simplifications here.
1. `l.stopCh` is already a `StopChan` with methods for creating contexts, so we we don't need to use these deprecated helpers from `package utils`
2. The `timeoutCtx` can be reused here, instead of creating a new one, since it is calibrated to the tick frequency already.
3. `timeoutCtx` can just be called `ctx` - maybe there used to be a name collision?

Reply:
Sounds good!